### PR TITLE
chore: run storybook migration + pnpm dedupe

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

The `@storybook/testing-library` got deprecated and threw warnings to run the migration script.

## Description

Ran the storybook migration script, all changes are automated from that but I tested that storybook still works as before.

### Double check
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/.storybook/main.cjs [hunk 1]
```diff
@@ -1,18 +1,20 @@
 const path = require('path');
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
Comment (madsnedergaard) [inline web/.storybook/main.cjs:3]: Do we want to support non ".stories.EXT" files?

Comment (madsnedergaard) [reply web/.storybook/main.cjs:3]: Also don't know why the two entries are not merged together 🤔

**Comment by madsnedergaard** [comment]
Nice, this upgrade also enables the new [interactive story generation](https://storybook.js.org/blog/interactive-story-generation/) :)

File: web/.storybook/main.cjs (hunk 1 above)
Comment (VIKTORVAV99) [reply web/.storybook/main.cjs:3]: Apparently they deprecated that according to the migration 🤷🏻‍♂️

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**